stack.yaml

```diff
@@ -3,16 +3,16 @@ resolver: lts-8.2
 packages:
   - '.'
   - location:
-      git: https://github.com/avieth/kademlia.git
-      commit: 90c94245d86e4e6008734ab1c7d1e070a75e4834
+      git: https://github.com/serokell/kademlia.git
+      commit: 21df94f41008f82ee023e8e0324c7e4a82c4fef2
     extra-dep: true
   - location:
       git: https://github.com/avieth/network-transport
       commit: f2321a103f53f51d36c99383132e3ffa3ef1c401
     extra-dep: true
   - location:
       git: https://github.com/avieth/network-transport-tcp
-      commit: ca42a954a15792f5ea8dc203e56fac8175b99c33
+      commit: eb1ed2fe4d4419c860ce060cb1091f7c8959134f
```

Comment:
Just a note: the old revision comes immediately after the new revision in `avieth/csl_2_nat` branch of network-transport-tcp. Cardano-sl needs to be built with `ca42a954` because it includes a (temporary) emergency patch which makes wallet/daedalus appear as unaddressable. It should be OK to use `eb1ed2fe` here though.